### PR TITLE
New package: python3-finvizfinance-0.14.5

### PR DESCRIPTION
**Pull request by jason1987d**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):

### CODE REVIEW

File: srcpkgs/python3-finvizfinance/patches/quote.py.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
+--- a/finvizfinance/quote.py	2023-02-03 21:17:45.000000000 -0600
```
Comment (Duncaen) [inline srcpkgs/python3-finvizfinance/patches/quote.py.patch:1]: Please add some context to the patch, like the source and what it tries to solve. You can put plain text for that at the start of the patch.

**Review by Duncaen — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-finvizfinance/patches/quote.py.patch (hunk 1 above)
Comment (Duncaen) [reply srcpkgs/python3-finvizfinance/patches/quote.py.patch:2]: Could you also link to https://github.com/lit26/finvizfinance/commit/ddd263736be278c8834242679c08c804d17ba1c0

Comment (jason1987d) [reply srcpkgs/python3-finvizfinance/patches/quote.py.patch:3]: Now it's failing when it worked earlier and I'm not sure why.

**Comment by jason1987d** [comment]
There actually appears to be a fairly new bug that I hadn't encountered when using this a month ago. I have found another python3 module that handles finviz that has a more recent update that I will take a look on.

**Pull request closed without merging**